templates/varnish.j2

```diff
@@ -97,7 +91,8 @@ VARNISH_TTL=120
 #
 # # DAEMON_OPTS is used by the init script.  If you add or remove options, make
 # # sure you update this section, too.
-DAEMON_OPTS="-a ${VARNISH_LISTEN_ADDRESS}:${VARNISH_LISTEN_PORT} \
+DAEMON_OPTS="{% for p in varnish_listen_ports %}-a :{{ p.port }}{% if p.proxy_proto is defined and p.proxy_proto and varnish_version|float >= 4.1 %},PROXY{% endif %} \
```

Comment:
I completely removed the variables `VARNISH_LISTEN_ADDRESS` and `VARNISH_LISTEN_PORT` because they are not compatible with a Varnish listening on multiple ports (by example `-a 81 -a 6081,PROXY`).
Instead, there is `for` loop directly into `DAEMON_OPTS` which add a `-a` for each ports defined in `varnish_listen_ports`.